screens/CameraScreen.tsx

```diff
@@ -0,0 +1,26 @@
+import { StackScreenProps } from "@react-navigation/stack";
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import { RootStackProps } from "../App";
+import CameraView from "../components/CameraView";
+
+type CameraProps = StackScreenProps<RootStackProps, "Camera">;
+
+const CameraScreen = ({ navigation, route }: CameraProps): JSX.Element => {
```

Comment:
`CameraView` should act like `CameraScreen` is acting here. Can we just use `CameraView` in place of this without creating a new component, or just have `CameraScreen` instead of `CameraView`? Same with `TranscriptionScreen`. I don't want to have both "Screen" and "View" because it'll get confusing

Reply:
I had used Screen over View because Screen indicates that it is NOT a component view. Also, it removes complexity from the files.

Reply:
The `View` components effectively _are_ the `Screen` components. Can we just use one and not both?

Reply:
sorry for any confusion - any component marked as a `View` was intended to mean that it would take up the entirety of the phone screen, so we should stick to using `View`s if we can.